bigquery/google/cloud/bigquery/dbapi/types.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Types used in the Google BigQuery DB-API.
+
+See `PEP 249`_ for details.
+
+.. _`PEP 249`:
```

Comment:
You don't want / need the backticks here? (Though maybe Sphinx is cool with it, I'm not sure.)